src/utils/finnFrontendMappe.ts

```diff
@@ -0,0 +1,27 @@
+import { existsSync } from 'fs';
+import path from 'path';
+
+const dirname = (): string => {
+    try {
+        return __dirname;
+    } catch (e) {
+        return path.dirname(new URL(import.meta.url).pathname);
+    }
+};
+
+export const finnProsjektMappe = (): string => {
+    const kjentFilIRotMappe = 'tsconfig.base.json';
+    let mappe = dirname();
+
+    while (!existsSync(path.join(mappe, kjentFilIRotMappe))) {
+        mappe = path.join(mappe, '..');
+    }
+
+    return mappe;
+};
+
+export const finnFrontendMappe = () => {
+    return path.join(finnProsjektMappe(), 'build');
+};
+
+export default finnFrontendMappe;
```

Comment:
Spennende! Hadde vært nice å ta en muntlig gjennomgang av hvordan dette henger sammen :D

Reply:
Kan godt det! Gikk kjapt igjennom det med Nora, er basically bare fordi build-mappa ligger to nivåer ned i strukturen mens den originale typescript-fila kun er ett nivå ned.